site/guide/set-up-model-workflows.qmd

```diff
@@ -0,0 +1,213 @@
+::: -->
+
+## Prerequisites
+
+- To customize workflows, you must hold the `Customer Admin` or `Developer` role
+- [Resource statuses](customize-resource-statuses.qmd) must be fully configured
+
+## Configure workflow steps
+
+To add or edit workflow steps:
+
+1. [Log in to the {{< var vm_platform >}}]({{< var vm_url >}}).
+
+2. In the left sidebar, click [**{{< fa gear >}} Settings**]({{< var vm_url >}}/settings).
+
+3. Under Workplace Settings, select **{{< fa arrow-right-arrow-left >}} Workflows**.
+
+4. Under available workflows, select the process you'd like to add or edit the steps for:
+ 
+    - To add a step, drag and drop a new step onto the canvas. Hover over the step until the **{{< fa ellipsis-vertical >}}** menu appears, then click on it and select **{{< fa gear >}} Configure**.
+    - To edit a step, hover over the step until the **{{< fa ellipsis-vertical >}}** menu appears. Then click on it and select **{{< fa gear >}} Configure**.
+
+5. After you're finished with step configuration, click **Save** to apply your changes before closing the configuration panel with **{{< fa x >}}**. 
+
+    ![](configure-steps.gif){fig-alt="A gif workflow step configuration."}
+
+    After you've configured a step, you can then [link your workflow together](#link-workflow-together).
+
+::: {.callout-tip appearance="minimal"}
+Refer to **[Available step types](#available-step-types)** for information on available step types and details on how to set up conditional requirements.
+:::
+
+### Available step types
+
+#### Status Change
+
+- Transitions a status. 
+- Requires [available resource statuses](customize-resource-statuses.qmd) associated with the workflow.
+
+| Field | Description |
+|---:|---|
+| When these conditions are met (optional) | [Add conditional requirements](#add-conditional-requirements) to qualify for this step.  |
+| Set status to (required) | Select the status to transition to. |
+: Status Change step configuration {.hover tbl-colwidths="[40,60]"}
+
+#### User Action
+- Creates a button that performs an action on behalf of the user. 
+- Requires [available resource statuses](customize-resource-statuses.qmd) associated with the workflow.
+
+| Field | Description |
+|---:|---|
+| Display action button when (optional) | [Add conditional requirements](#add-conditional-requirements) for displaying this step.  |
+| Action button label (required) | Text that displays on your action button. |
+| Request fields (optional) | Include [custom fields](manage-inventory-custom-fields.qmd) alongside your action if desired. Select the field, then click **+ Add Field** to insert.  |
+| To set status (required) | Select the [status](customize-resource-statuses.qmd) to transition to. |
+: User Action step configuration {.hover tbl-colwidths="[40,60]"}
+
+::: {.grid}
+
+::: {.g-col-7}
+![](configure-user-action.png){fig-alt="A screenshot showing a User Action step configuration with a request field of Date On Hold."}
```

Comment:
The screenshots are really helpful for this rather complex task, but they sometimes show extra stuff that adds visual complexity. E.g. here, I'd remove the extraneous stuff outside the modal/popup itself. 

Related, we should have a chat about borders around images. We don't have a standard for this and sometimes I've been adding them manually and sometimes with some CSS. In this PR, images don't have borders but they might benefit from them — ideally we sort this out globally via `styles.css` so they work for all images without having to edit them individually (other than removing manually added borders from existing images).

Reply:
I did already crop this screenshot, and I don't think making it smaller helps here — as a user, I'd expect to see the field in context with the other stuff so I could potentially locate it instead of looking for an element in isolation.

Reply:
It's stuff like this that I am talking about around the edges which doesn't add context, it just looks like the screenshot needs to be edited. I would crop this and other screenshots to be just the modal itself, not the stray background lines or text behind it. 

<img width="536" alt="image" src="https://github.com/validmind/documentation/assets/15148011/02dc4a63-3e5f-4e03-bdeb-266aad32c0a7">

Reply:
Ohhh I see, sorry, I thought you meant the rest of the configuration box. I can definitely make the crop tighter.